### ISSUE TITLE
not able to read property file normally.

### ISSUE THREAD

**Issue by juneng603** [Impetus/kundera#94]
The class HBasePropertyReader has read().

in this method, it try to load "kundera-hbase.properties" for hbase.

but, it is not able to load it properly.

``` java
# propertyName = "kundera-hbase.properties which is located in "src/main/resources/"
ClassLoader.getSystemResourceAsStream(propertyName)
```

it returns always "null" value.

if I modified it to 

``` java
# propertyName = "kundera-hbase.properties which is located in "src/main/resources/"
#ClassLoader.getSystemResourceAsStream(propertyName)
HBasePropertyReader.class.getResourceAsStream(propertyName)
```

it returns a stream object what I want to have.

**Comment by mevivs** [comment]
Sounds like a bug.

Can you please try it with

```
InputStream inStream = propertyName != null ? Thread.currentThread().getContextClassLoader().getResourceAsStream(propertyName) : null;
```

**Comment by juneng603** [comment]
a way you mentioned is working nicely. ;)

anyway, in the wiki, there are pages to set up configurations for hbase. 

is it working?

**Comment by mevivs** [comment]
Okay.This will be fixed in latest trunk branch. Yes, those are definitly working. There is one HBasePropertyReaderTest added   to demonstrate the same.

Plese close this issue, if it is working fine for you.

-Vivek

**Comment by juneng603** [comment]
wow, yes, it is working very perfectly.

in my debug, the lastest library 2.0.7 also returns normal property name which is named as "kundera-hbase.properties".

but, it got a "null" value from ClassLoader.

Can I ask you what the differences between the test-case and my project?

I mean that why the case I have return a "null" value.

**Comment by mevivs** [comment]
ClassLoader.getSystemResourceAsStream is trying to load system resources but your properties file is not system resources. 

Other alternative, which i have suggested is a better way to load user as well as system resources.

-Vivek

**Comment by juneng603** [comment]
Is there any way to put my configuration files into a system resources area?

thanks for your speedy and kind answer

**Comment by mevivs** [comment]
Not sure. But this should give you a brief about this :

http://stackoverflow.com/questions/2815404/load-properties-file-in-jar

**Comment by mevivs** [comment]
Shall i close this now, if working fine for you?

-Vivek

**Comment by juneng603** [comment]
thanks for your nice supports.

additionally, 
currently, I couldn't load my property because of this problem on release 2.0.7.

Is there any workaround way to fix for my one?

**Comment by mevivs** [comment]
There is a branch(2.0.7.1) created for bug fixes done on 2.0.7. You need to download and build kundera-hbase from this branch to get this working.
Alternatively, if you have download 2.0.7 source code then you can manually merge commit made for 1eb7676dc1f57c7fd62876c0414e4633618b3bcd (not a preferable way, though)

-Vivek

**Comment by juneng603** [comment]
thanks for your responses again.
